### ISSUE TITLE
Using createMissingTablesAndColumns() generates a spurious WARN message in log files

### ISSUE THREAD

**Issue by bberliner** [JetBrains/Exposed#480]
Whenever I use `createMissingTablesAndColumns()`, I see a WARN log message emitted that should either not be emitted (because it is wrong) or `create()` is not properly creating all the indexes required (?). Using the example app from the Wiki and adding a call to `createMissingTablesAndColumns` immediately after the table has been created:
```
import org.jetbrains.exposed.dao.IntIdTable
import org.jetbrains.exposed.sql.*
import org.jetbrains.exposed.sql.SchemaUtils.create
import org.jetbrains.exposed.sql.SchemaUtils.createMissingTablesAndColumns
import org.jetbrains.exposed.sql.transactions.transaction

fun main(args: Array<String>) {
    //an example connection to H2 DB
    Database.connect("jdbc:h2:mem:test;DB_CLOSE_DELAY=-1", driver = "org.h2.Driver")

    transaction {
        // print sql to std-out
        addLogger(StdOutSqlLogger)
        create(Cities)
        createMissingTablesAndColumns(Cities)

        // insert new city. SQL: INSERT INTO Cities (name) VALUES ('St. Petersburg')
        val stPeteId = Cities.insert {
            it[name] = "St. Petersburg"
        } get Cities.id

        // 'select *' SQL: SELECT Cities.id, Cities.name FROM Cities
        println("Cities: ${Cities.selectAll()}")
    }
}

object Cities : IntIdTable() {
    val name = varchar("name", 50)
}
```
We see the following output in the logs:
```
13:56:34.856 [main] DEBUG Exposed - CREATE TABLE IF NOT EXISTS CITIES (ID INT AUTO_INCREMENT PRIMARY KEY, NAME VARCHAR(50) NOT NULL)
SQL: CREATE TABLE IF NOT EXISTS CITIES (ID INT AUTO_INCREMENT PRIMARY KEY, NAME VARCHAR(50) NOT NULL)
13:56:34.867 [main] INFO Exposed - Preparing create tables statements took 6ms
13:56:34.867 [main] INFO Exposed - Executing create tables statements took 0ms
13:56:34.911 [main] INFO Exposed - Extracting table columns took 44ms
13:56:34.912 [main] INFO Exposed - Extracting column constraints took 1ms
13:56:34.912 [main] INFO Exposed - Preparing alter table statements took 45ms
13:56:34.912 [main] INFO Exposed - Executing alter table statements took 0ms
13:56:34.916 [main] WARN Exposed - Indices exist in database and not mapped in code on class 'CITIES':
13:56:34.917 [main] WARN Exposed - 		Unique Index 'CITIES_ID_UNIQUE' for 'CITIES' on columns Cities.id
13:56:34.917 [main] INFO Exposed - Checking mapping consistence took 5ms
13:56:34.924 [main] DEBUG Exposed - INSERT INTO CITIES (NAME) VALUES ('St. Petersburg')
SQL: INSERT INTO CITIES (NAME) VALUES ('St. Petersburg')
Cities: org.jetbrains.exposed.sql.Query@6f6745d6
```
I believe that `create()` is properly creating the tables and indexes, which leads me to believe that the WARN message is wrong or not needed.

**Comment by Tapac** [comment]
Possibly fixed in #498

**Comment by lncubus** [comment]
> Possibly fixed in #498

No, I experience this issue with H2 database

**Comment by SinonCute** [comment]
Still having this issue with H2
